tests/console/wavpack.pm

```diff
@@ -72,7 +76,11 @@ sub run {
     # unregister SDK
     if (is_sle) {
         if (get_var('BETA')) {
-            zypper_call "rr SDK";
+            if (is_sle('15-sp1+')) {
+                remove_suseconnect_product('sle-module-desktop-applications');
+            } else {
+                zypper_call "rr SDK";
```

Comment:
I think this line can be deleted, as the `get_addon_fullname('sdk')` should give the right name
```suggestion
                remove_suseconnect_product(get_addon_fullname('sdk'));
```

Reply:
I think this repo is added manually:
```perl
zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => "SDK";
```

remove_suseconnect_product will fail